pom.xml

```diff
@@ -86,6 +86,11 @@
 			<artifactId>opentelemetry-logback-appender-1.0</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.opentelemetry.instrumentation</groupId>
+			<artifactId>opentelemetry-apache-httpclient-5.2</artifactId>
+			<version>2.3.0-alpha</version>
```

Comment:
this version can be omissed? the bom is already imported

Reply:
Right, thanks!